hbase-client/src/main/java/org/apache/hadoop/hbase/client/HBaseAdmin.java

```diff
@@ -1911,7 +1916,13 @@ public void split(final TableName tableName, final byte[] splitPoint) throws IOE
         continue;
       }
       // call out to master to do split now
-      splitRegionAsync(r, splitPoint);
+      try {
+        splitRegionAsync(r, splitPoint).get();
```

Comment:
`split` will return immediately without waiting for or surfacing any exception from the operation. This seems wrong. Yet, it is a semantic change over previously released versions. For discussion.

Reply:
Should be the same with mergeRegions method.

Reply:
Same, will ignore.